pages/data-design/avo-tracking-plan/journeys.mdx

```diff
@@ -0,0 +1,233 @@
+- For the first and subsequent triggers on a journey step, you can use the AI menu above the step to generate triggers with an optional custom prompt. 
+
+For a full walkthrough of building journeys on the canvas, see [Creating a journey](#creating-a-journey). Learn more about pricing for AI actions in [Agentic data design](/data-design/avo-tracking-plan/agentic-data-design).
+
+### Journey trigger property conditions
+
+A journey trigger property condition is an optional condition for each [event](/data-design/avo-tracking-plan/events) or [event variant](/data-design/avo-tracking-plan/event-variants) connected to the trigger, highlighting the properties that are important or have specific values that should be sent in that scenario.
+
+#### Property condition building blocks
+
+A property condition is composed of the following details:
+- [Property](/data-design/avo-tracking-plan/journeys#properties-in-property-conditions): The property that the condition is for
+- [Operator](/data-design/avo-tracking-plan/journeys#property-conditions-operators): The operator defining the condition that the property should meet
+- [Value](/data-design/avo-tracking-plan/journeys#property-conditions-values): The value that the property should meet in this trigger for this journey step
+- [Property condition note](/data-design/avo-tracking-plan/journeys#property-conditions-notes): An optional note for the property condition, with context about the property and the condition
+
+##### Properties in property conditions
+
+For each event or variant in a trigger, you can add properties to define the property condition. When you click "Add property condition" in the journey trigger details you get a list of the following options:
+- **Existing properties on the event or variant**: That means you pick a property from the list of properties that have already been added to the event or variant to highlight and add a condition for it
+- **Existing properties not yet added to the event or variant**: That means you pick a property from the list of properties in the tracking plan that have not yet been added to the event or variant and add it to the event or variant
+- **New properties**: That means you can create a new property to both the tracking plan and the event or variant
+
+##### Property condition operators
+
+Available operators for string and boolean properties are:
+- **is**: The property value should be equal to the value(s) specified
+  - If *(not set)* is selected and the property is optional, then the property continues being optional. If it should have a value, then it's one of the specified ones.
+  - If *(not set)* is deselected, then the property is required and must have a value.
+- **is not**: The property value should not be equal to the value(s) specified
+  - If *(not set)* is selected and the property is optional, then the property continues being optional. If it should have a value, then it's not one of the specified ones.
```

Comment:
should we say "sometimes sent" here rather than optional, to match the words we use for that in the product?

Reply:
done